sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusMessageSerializer.java

```diff
@@ -67,6 +73,9 @@
 
     private final ClientLogger logger = new ClientLogger(ServiceBusMessageSerializer.class);
 
+    private static final Long ONE_TICK_SIZE = 10000000L;
+    private static final Long ONE_NANO_SIZE = 1000000000L;
```

Comment:
TICK_PER_SECOND ?

Reply:
> TICK_PER_SECOND ?

Yeah, that's much easier to understand. But I'm not sure it's easy to understand the follow function. I tried to avoid magic number, but it's hard to describe what the function do.

Reply:
Is there documentation you can link to as to where you got this value?